ios/CoinbaseWalletSDK/Provider/MobileWalletProvider.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  WalletProvider.swift
+//  CoinbaseWalletSDK
+//
+//  Created by Amit Goel on 11/4/22.
+//
+
+public protocol MobileWalletProviderProtocol {
+    func getWallets() -> [Wallet]
+}
+
+public class MobileWalletProvider : MobileWalletProviderProtocol {
```

Comment:
what is this protocol for? can it be just
```suggestion
public class MobileWalletProvider {
```

Reply:
actually, let's just remove this class and use `Wallet.defaultValues`